### DISCUSSION TITLE
Issues accessing GitLab repositories

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#1282] [Announcements]
If you’re facing errors on accessing your GitLab repositories in Codemagic, such as `invalid_grant` or `400 Client Error: Bad Request`, we advise you to reconnect the GitLab integration to update the OAuth token.
If you use a GitLab account to log in to Codemagic, please log out and log in again. If you’re using the GitLab integration in a team, please disconnect and reconnect the integration in team settings. 
Should the issue persist after these steps, do not hesitate to get in touch with us.

**Comment by AnkitSimform** [comment]
Hello, 

I am facing an issue in fetching the branch list to start to build manually. I have logged in using GitLab but not facing the issue you mentioned above. Could you please guide me here?

**Comment by cpuell** [comment]
Disconnect the Account in CM and reconnect did work for me. Using GitLab.
